exercises/Phil_Colebrooke/ex01_basics/src/main.cpp

```diff
@@ -18,16 +12,18 @@ int main(int argc, char *argv[]) {
     return 1;
   }
 
-  inputFilename = argv[1];
-  outputFilename = argv[2];
+  std::string inputFilename{argv[1]};
+  std::string outputFilename{argv[2]};
 
   // the inputted min word length is converted to int
-  minWordLength = atoi(argv[3]);
+  int minWordLength{atoi(argv[3])};
 
+  bool success{false};
+  std::map<std::string, int> wordsAndCounts;
   success = countWords(inputFilename, wordsAndCounts, minWordLength);
 
   if (success) {
-    sorted = orderByUsage(wordsAndCounts);
+    auto sorted{orderByUsage(wordsAndCounts)};
```

Comment:
`sorted` is an OK name, but it could be a little more descriptive. What exactly is sorted here? Container names should ideally describe what they contain